### PR TITLE
Bhijn's touching dogborg balance again everyone panic

### PR DESCRIPTION
**Pull request by deathride58**
# RE:TELEGRAPHING: http://www.strawpoll.me/14162396/r Vote in this if you want telegraphing to stay or go.

:cl: deathride58
balance: Sec doggos have a disabler again
balance: Sec doggos need 1 more second for their leg actuators to recharge after a pounce.
balance: Sec doggos no longer have a nuke pinpointer when emagged
balance: Sechounds stun for half a second longer
balance: Sechound pounce spoolup has been reduced from 0.5 seconds to 0.3 seconds.
balance: Scrubpups now have actual working mops on their feet instead of cheap plastic imitations
balance: Scrubpups can eat normal-sized items again
balance: Scrubpups can only carry 20 items now
balance: Scrubpups no longer have flashes
/:cl:

### CODE REVIEW

File: code/citadel/dogborgstuff.dm [hunk 1]
```diff
@@ -356,7 +356,15 @@
 		/obj/item/documents/syndicate,
 		/obj/item/disk/nuclear,
 		/obj/item/bombcore,
-		/obj/item/grenade)
+		/obj/item/grenade,
+		/obj/item/melee,
```
Comment (Poojawa) [inline code/citadel/dogborgstuff.dm]: why

File: code/citadel/dogborgstuff.dm [hunk 2]
```diff
@@ -356,7 +356,15 @@
...
+		/obj/item/key,
+		/obj/item/kitchen/knife,
+		/obj/item/storage,
```
Comment (Poojawa) [inline code/citadel/dogborgstuff.dm]: Literally what

File: code/citadel/dogborgstuff.dm [hunk 3]
```diff
@@ -356,7 +356,15 @@
...
+		/obj/item/restraints,
+		/obj/item/clockwork,
+		/obj/item/device/pda,
+		/obj/item/device/assembly/flash)
```
Comment (Poojawa) [inline code/citadel/dogborgstuff.dm]: be Janidog, be unable to do anything to defend yourself from anything if hacked.

seems legit.

**Comment by LetterJay** [comment]
Ready for merge?

**Review by Poojawa — APPROVED** [review]

**Comment by deathride58** [comment]
👍

**Comment by davethwave** [comment]
Scrub puppies should keep their flash though since you know... borg eyes = flash

**Comment by deathride58** [comment]
Scrubpups losing their flash keeps it consistent with the other dogborgs, which don't have flashes at all.

**Comment by davethwave** [comment]
Oh, I never noticed.

**Pull request merged**